Munny_Lamp/Munny_Lamp.ino

```diff
@@ -0,0 +1,217 @@
+                              running this at least once is a good idea.
+   
+                              When deploying your project, however, you will
+                              want to disable factory reset by setting this
+                              value to 0.  If you are making changes to your
+                              Bluefruit LE device via AT commands, and those
+                              changes aren't persisting across resets, this
+                              is the reason why.  Factory reset will erase
+                              the non-volatile memory where config data is
+                              stored, setting it back to factory default
+                              values.
+       
+                              Some sketches that require you to bond to a
+                              central device (HID mouse, keyboard, etc.)
+                              won't work at all with this feature enabled
+                              since the factory reset will clear all of the
+                              bonding data stored on the chip, meaning the
+                              central device won't be able to reconnect.
+    PIN                       Which pin on the Arduino is connected to the NeoPixels?
+    NUMPIXELS                 How many NeoPixels are attached to the Arduino?
+    -----------------------------------------------------------------------*/
+    #define FACTORYRESET_ENABLE     0
+/*=========================================================================*/
+
+
+// Create the bluefruit object, either software serial...uncomment these lines
+Adafruit_BluefruitLE_SPI ble(BLUEFRUIT_SPI_CS, BLUEFRUIT_SPI_IRQ, BLUEFRUIT_SPI_RST);
+
+// A small helper
+void error(const __FlashStringHelper*err) {
+  Serial.println(err);
+  while (1);
+}
+
+// function prototypes over in packetparser.cpp
+uint8_t readPacket(Adafruit_BLE *ble, uint16_t timeout);
+float parsefloat(uint8_t *buffer);
+void printHex(const uint8_t * data, const uint32_t numBytes);
+
+// the packet buffer
+extern uint8_t packetbuffer[];
+
+
+/**************************************************************************/
+/*!
+    @brief  Sets up the HW an the BLE module (this function is called
+            automatically on startup)
+*/
+/**************************************************************************/
+void setup(void)
+{
+  delay(500);
+  pinMode(POWER_PIN, OUTPUT);
+  digitalWrite(POWER_PIN, HIGH);
+  pinMode(RED_LED, OUTPUT);
+  pinMode(GREEN_LED, OUTPUT);
+  pinMode(BLUE_LED, OUTPUT);
+
+  analogWrite(RED_LED, 0);
+  analogWrite(GREEN_LED, 0);
+  analogWrite(BLUE_LED, 255);  // startup color, waiting for BLE connection
+
+  if (! lis.begin(0x18)) {   // change this to 0x19 for alternative i2c address
+    Serial.println("Couldnt start LIS3DH");
+    while (1);
+  }
+
+  Serial.begin(115200);
+  Serial.println(F("Adafruit Bluefruit MUNNY Neopixel Color Picker"));
```

Comment:
fix

Reply:
Oh, is it to fix the name since it isn't the NeoPixel we're using? Got it, thanks.

Reply:
yah is not neopixels!